CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to the "branch-creator" extension will be documented in this
 
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
 
+## [0.0.9] - YYYY-MM-DD
```

Comment:
NICE!

Reply:
The date is a placeholder until the release date.